enterprise-suite/tests/smoke_prometheus

```diff
@@ -9,9 +9,20 @@ source smokecommon
 #
 
 set -e
+NAMESPACE="smoketest-prometheus"
+kubectl create namespace $NAMESPACE
 for app in resources/app*.yaml; do
-  kubectl apply -f "$app" -n lightbend
+  kubectl apply -f "$app" -n $NAMESPACE
 done
+
+cleanup() {
+  for app in resources/app*.yaml; do
+    kubectl delete -f "$app" -n $NAMESPACE
+  done
+  kubectl delete namespace $NAMESPACE
+}
+
+trap cleanup 0
```

Comment:
Is there a reason to create it in a separate namespace to where we install the console? This is okay on minikube, but on openshift or more restricted environments that control namespace creation it could be problematic. I think it's better to keep the apps inside of the console namespace.

Reply:
Ah good point. I was hoping to "delete namespace" and clean everything up after a test, but deleting things turned up yet new edge cases so still pondering how to best clean up the test.